static/src/javascripts/projects/common/modules/experiments/tests/football-weekly-treat-vs-container.js

```diff
@@ -0,0 +1,186 @@
+import containerWaveTablet from 'svgs/journalism/football-weekly-container/waveform-tablet.svg';
+import containerWaveMobile from 'svgs/journalism/football-weekly-container/waveform-mobile.svg';
+import containerWaveTiny from 'svgs/journalism/football-weekly-container/waveform-tiny.svg';
+import containerButton from 'svgs/journalism/football-weekly-container/play-btnbig.svg';
+
+const getHeadlineText = (el: Element): string =>
+    el.textContent.split(/[-–]/)[0];
+
+const runTreatTest = (): void => {
+    // Get the headline from podcast container and insert the test treat
+    const podcastContainer = document.querySelector(
+        '.facia-page #world-cup-daily'
+    );
+    const headlinesContainer = document.querySelector('.facia-page #headlines');
+
+    if (headlinesContainer && podcastContainer) {
+        const headline = podcastContainer.querySelector('a.js-headline-text');
+        if (headline) {
+            const headlineText = getHeadlineText(headline);
+            const url = headline.getAttribute('href');
+
+            if (url) {
+                const newTreat = template(treatHtml, {
+                    headline: headlineText,
+                    url: `${url}?CMP=football-weekly-treat`,
+                    logo: logo.markup,
+                    button: addClassesAndTitle(button.markup, [
+                        'football-weekly-treat__player-button',
+                    ]),
+                    waveSmall: addClassesAndTitle(waveSmall.markup, [
+                        'football-weekly-treat__player-wave',
+                        'football-weekly-treat__player-wave-small',
+                    ]),
+                    waveLarge: addClassesAndTitle(waveLarge.markup, [
+                        'football-weekly-treat__player-wave',
+                        'football-weekly-treat__player-wave-large',
+                    ]),
+                    waveMedium: addClassesAndTitle(waveMedium.markup, [
+                        'football-weekly-treat__player-wave',
+                        'football-weekly-treat__player-wave-medium',
+                    ]),
+                });
+
+                const breakpoint = getBreakpoint(true);
+                if (breakpoint === 'leftCol' || breakpoint === 'wide') {
+                    // In the headlines container's treat spot
+                    const html = `<div>${newTreat}</div>`;
+
+                    const treats = headlinesContainer.querySelector(
+                        '.treats__container'
+                    );
+                    if (treats) {
+                        treats.innerHTML = html;
+                    }
+                } else {
+                    // As a new section under headlines
+                    const html = `<section class='fc-container football-weekly-treat__container'><div class='fc-container__inner'>${newTreat}</div></section>`;
+                    headlinesContainer.insertAdjacentHTML('afterend', html);
```

Comment:
Nice 👍

Reply:
DOM mutations should probably go inside a `fastdom.write` thunk for good measure